gemini/use-cases/retrieval-augmented-generation/rag_evaluation/ragas_with_gemini.ipynb

```diff
@@ -0,0 +1,816 @@
+    "    </a>\n",
+    "  </td>\n",
+    "  <td style=\"text-align: center\">\n",
+    "    <a href=\"https://console.cloud.google.com/vertex-ai/workbench/deploy-notebook?download_url=https://raw.githubusercontent.com/GoogleCloudPlatform/generative-ai/blob/main/language/rag_evaluation/ragas_with_gemini.ipynb\">\n",
+    "      <img src=\"https://lh3.googleusercontent.com/UiNooY4LUgW_oTvpsNhPpQzsstV5W8F7rYgxgGBD85cWJoLmrOzhVs_ksK_vgx40SHs7jCqkTkCk=e14-rj-sc0xffffff-h130-w32\" alt=\"Vertex AI logo\"><br> Open in Vertex AI Workbench\n",
+    "    </a>\n",
+    "  </td>\n",
+    "</table>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "CkHPv2myT2cx"
+   },
+   "source": [
+    "## Overview\n",
+    "\n",
+    "### [RAGAS](https://docs.ragas.io/en/stable/)\n",
+    "\n",
+    "Ragas is a framework that helps you evaluate your Retrieval Augmented Generation (RAG) pipelines. RAG denotes a class of LLM applications that use external data to augment the LLM’s context. There are existing tools and frameworks that help you build these pipelines but evaluating it and quantifying your pipeline performance can be hard. This is where Ragas (RAG Assessment) comes in.\n",
+    "\n",
+    "### Gemini\n",
+    "\n",
+    "Gemini is a family of generative AI models developed by Google DeepMind that is designed for multimodal use cases. The Gemini API gives you access to the Gemini Pro Vision and Gemini Pro models.\n",
+    "\n",
+    "### Vertex AI Gemini API\n",
+    "\n",
+    "The Vertex AI Gemini API provides a unified interface for interacting with Gemini models. There are currently two models available in the Gemini API:\n",
+    "\n",
+    "- **Gemini Pro model** (`gemini-pro`): Designed to handle natural language tasks, multiturn text and code chat, and code generation.\n",
+    "- **Gemini Pro Vision model** (`gemini-pro-vision`): Supports multimodal prompts. You can include text, images, and video in your prompt requests and get text or code responses.\n",
+    "\n",
+    "You can interact with the Gemini API using the following methods:\n",
+    "\n",
+    "- Use the [Vertex AI Studio](https://cloud.google.com/generative-ai-studio) for quick testing and command generation\n",
+    "- Use cURL commands\n",
+    "- Use the Vertex AI SDK\n",
+    "\n",
+    "This notebook focuses on using the **Gemini model with RAGAS**\n",
+    "\n",
+    "For more information, see the [Generative AI on Vertex AI](https://cloud.google.com/vertex-ai/docs/generative-ai/learn/overview) documentation.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "DrkcqHrrwMAo",
+    "tags": []
+   },
+   "source": [
+    "### Objectives\n",
+    "\n",
+    "In this notebook we will focus on using the Vertex AI Gemini API with RAGAS\n",
+    "We will use the Gemini Pro (`gemini-pro`) model for Q&A evaluation.\n",
```

Comment:
Change this to have the full name (Gemini 1.0 Pro or Gemini 1.5 Pro)

Reply:
Added the Specific model version  full name